### ISSUE TITLE
Can't search Dataverse 5.0 guides

### ISSUE THREAD

**Issue by pdurbin** [IQSS/dataverse#7228]
If I go to http://guides.dataverse.org and try to do a search of the Dataverse 5.0 Guides, it fails.

Here are some screenshots of a search for the term metadata with the Javascript console open.

<img width="1392" alt="Screen Shot 2020-08-26 at 10 53 08 AM" src="https://user-images.githubusercontent.com/21006/91319588-8c165b80-e78a-11ea-9854-f406e4d7ca05.png">
<img width="1392" alt="Screen Shot 2020-08-26 at 10 53 44 AM" src="https://user-images.githubusercontent.com/21006/91319595-8de01f00-e78a-11ea-955e-370831c053ac.png">
<img width="1392" alt="Screen Shot 2020-08-26 at 10 53 54 AM" src="https://user-images.githubusercontent.com/21006/91319600-8f114c00-e78a-11ea-912b-87c3f30a8f3b.png">

Going to http://guides.dataverse.org/en/5.0/ instead of "latest" doesn't help.

Search works fine at http://guides.dataverse.org/en/4.20 . The expected results for "metadata" are something like this:

<img width="1286" alt="Screen Shot 2020-08-26 at 10 56 14 AM" src="https://user-images.githubusercontent.com/21006/91319832-d697d800-e78a-11ea-87ac-995dbab9fce9.png">

Search also works fine if I build the docs on my laptop as long as I host the docs with a web server such as `python3 -m http.server`.

**Comment by djbrooke** [comment]
@pdurbin @donsizemore @mheppler @kcondon thanks for discussion and troubleshooting this in various ways today and thanks @pdurbin for putting up the build. The search is now working so I'm going to close this. I created #7233 as a follow on here.

**Comment by pdurbin** [comment]
> Search also works fine if I build the docs on my laptop

I assumed everyone is using the same version of Sphinx everywhere. On my laptop I was using Sphinx 1.5.6 because that's what's in doc/sphinx-guides/requirements.txt

It turns out, Dataverse 5.0 was built with Sphinx 3.2.1 and this version didn't work for me locally (though I see there's pull request #7232) that might fix it.

@djbrooke @kcondon @mheppler @donsizemore and I had a call about all this and we decided that I would build the HTML version of the 5.0 docs locally using Sphinx 1.5.6, where search works and that we've used for 4.20, 4.19, etc. Then copy those files into place on the guides server and copy over the ePub and PDF from the 3.2.1 build where search isn't working. It turns out there was no PDF to copy over but the rest is done ~~so I'm moving this to QA~~.

Danny and I both observed that if you've already been looking at the 5.0 docs, you'll have to switch to an icognito window or clear your cache for search to work.

Longer term, we'll make sure we can use newer versions of Sphinx in #7233.
